src/index.js

```diff
@@ -132,8 +132,8 @@ function checkForTaskKill(ecs, service, options) {
   if (options.killTask) {
     if (options.verbose) console.info('searching for running task to stop')
     return ecs.listTasks({ cluster: options.cluster, serviceName: options.service }).promise().then(res => {
-      if (res.data && res.data.taskArns && res.data.taskArns.length) {
-        const task = res.data.taskArns[0]
+      if (res.taskArns && res.taskArns.length) {
```

Comment:
why not `Array.isArray(res.taskArns)`?

Reply:
I didn't really change the logic after forking the repo. But `Array.issArray` would behave differently when the array is empty? Right now the `if` does not get executed when the array is empty. When changing to `isArray` it would get executed.

Reply:
youre right, then lets leave it like this